src/redux/categories/categories.js

```diff
@@ -0,0 +1,17 @@
+const CHECK_STATUS = 'CHECK_STATUS';
```

Comment:
- [ ] As requested by the previous reviewer, Please implement the ducks' pattern value for categories consts as well.

`const CHECK_STATUS = 'bookstore/categories/CHECK_STATUS'.`